ocrd/ocrd/processor/base.py

```diff
@@ -123,3 +125,48 @@ def input_files(self):
                 self.input_file_grp
                 ))
         return ret
+
+    @property
```

Comment:
```suggestion

```

If it is a property, it shouldn't have arguments.

Reply:
Oh, sorry, forgot to remove that when I added `require_first`.